src/reducers/realTime.js

```diff
@@ -39,7 +51,20 @@ const realTime = (state = initialState, action) => {
 
     case SET_CONNECTED_PEERS:
         return Object.assign({}, state, {
-            connectedPeers: action.payload
+            connectedPeers: action.payload,
+            golemStatus: {
+                status: 'Ready',
```

Comment:
Could use a ENUM to not hard-code the status strings, not trivial, maybe follow up issue?